cds_ils/importer/providers/cds/rules/base.py

```diff
@@ -811,12 +811,13 @@ def imprint(self, key, value):
     try:
         date = parser.parse(clean_val("c", value, str, req=True))
     except ParserError:
+        date = None
```

Comment:
not sure why this since it is raising, but not very important I guess

Reply:
👀 that was a leftover, thanks!